Src/Particle/AMReX_ParticleTile.H

```diff
@@ -663,7 +699,10 @@ struct ParticleTile
         ArrayOfStructs<ParticleType, Allocator>>::type;
     //using ParticleVector = typename AoS::ParticleVector;
 
-    using SoA = StructOfArrays<NArrayReal, NArrayInt, Allocator>;
+    using SoA = typename std::conditional<
+        ParticleType::is_soa_particle,
+        StructOfArrays<NArrayReal, NArrayInt, Allocator, true>,
+        StructOfArrays<NArrayReal, NArrayInt, Allocator, true>>::type;
```

Comment:
@atmyers isn't this in both cases `true` now - so essentially we can remove the extra template argument in `StructOfArrays`?

Reply:
This is a typo, I believe the second should be `false`...